vm/src/stdlib/sys.rs

```diff
@@ -701,7 +701,7 @@ mod sys {
         /// -X dev
         dev_mode: bool,
         /// -X utf8
-        utf8_mode: u8,
+        utf8_mode: bool,
```

Comment:
I am sorry for wrong guide. This is `u8` on this struct

Reply:
Hehe, no problemo.